content/docs/api-reference/params_show.md

```diff
@@ -0,0 +1,198 @@
+# dvc.api.params_show()
+
+Load <abbr>parameters</abbr> (name and values) tracked in a <abbr>DVC
+project</abbr>.
+
+```py
+def params_show(
+    *targets: str,  # Optional
+    stages: Optional[Union[str, Iterable[str]]] = None,
+    repo: Optional[str] = None,
+    rev: Optional[str] = None,
+    deps: bool = False,
+) -> Dict:
+```
+
+## Usage:
+
+```py
+import dvc.api
+
+params = dvc.api.params_show()
+```
+
+## Description
+
+Retrieves <abbr>params</abbr> keys and values from a <abbr>DVC project</abbr>
+and returns a dictionary, such as:
+
+```json
+{
+  "split": 0.2,
+  "seed": 20170428
+}
+```
+
+Without arguments, this function will retrieve all params from all tracked
+parameter files (used in any `dvc.yaml` file). This applies to the current
+project version when using Git (including any changes in the working tree).
+
+The function parameters (below) let you restrict what's retrieved.
+
+## Parameters
+
+- `*targets` - paths to one or more valid parameter file(s) to retrieve params
+  from. For example, `"params.py, myparams.toml"`. If no `targets` are provided,
```

Comment:
```suggestion
  from. For example, `"params.py", "myparams.toml"`. If no `targets` are provided,
```

TBH I would suggest removing that sentence. I don't see similar examples for targets in the CLI, and I think people should know how Python positional args work?

Reply:
PTAL